orbitize/sampler.py

```diff
@@ -225,7 +225,8 @@ def __init__(self, system, like='chi2_lnlike', custom_lnlike=None, use_c=True, u
             self.system,
             sampler_name=self.__class__.__name__,
             post=None,
-            lnlike=None
+            lnlike=None,
+            version_number=orbitize.__version__
```

Comment:
I originally made it an input to Results so that when we're restoring an object that was created with an older version, we're able to pass in the version number saved with the Results object. Is there a better way to do that with a global variable?

Reply:
Oh sorry, I misunderstood how this was used. This is good!